README.md

```diff
@@ -227,9 +225,10 @@ You can specify command line arguments (without any prefix) to switch between re
 <br/>
 
 ### Other software using Wicked Engine
-- <a href="https://www.game-guru.com/max">Game Guru MAX</a>: Easy to use game creator software
-- <a href="https://www.youtube.com/watch?v=0SxXmnSQ6Q4">Flytrap</a>: Demoscene production by qop
-- Your project: add your project to this readme and open a pull request
+- <a href="https://github.com/MolassesLover/WickedEngine-Demo.git">The Wicked Engine demo</a>: A fully-fledged game in the works.
```

Comment:
This should be added later if the project is a bit more matured, and it should be added to the end of the list.